app/Helpers/database/player-game.php

```diff
@@ -114,6 +114,10 @@ function testBeatenGame(int $gameId, string $user, bool $postBeaten): array
                 awardDate: Carbon::parse(calculateBeatenGameTimestamp($userAchievements)),
                 displayOrder: 0
             );
+
+            if ($isBeatenHardcore) {
+                static_addnewhardcoregamebeaten($gameId, $user);
+            }
```

Comment:
I assume it's because not all games have their Beaten conditions defined yet, but it's weird that Beaten is so much lower than Mastered:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/c9f952fa-b166-4d2f-9d09-65cd6666a7a6)

Reply:
Your assumption is correct. A handful of devs are diligently working through defining those conditions right now, but there is still a lot of work to be done in that regard.